testutils/ocispec.go

```diff
@@ -0,0 +1,219 @@
+package testutils
+
+import (
+	"github.com/docker/libentitlement/types"
+	"github.com/opencontainers/runtime-spec/specs-go"
+	"reflect"
+)
+
+var (
+	testCapslist = mobyDefaultCaps
```

Comment:
I don't think this is used, should it be removed?  As discussed, it would be good to use `mobyDefaultCaps` in tests such that we check an entitlement's cap list is:

`mobyDefaultCaps + entitlement.capsToAdd - entitlement.capsToRemove`